### PR TITLE
HL-12 | Enable authentication

### PR DESCRIPTION
**Pull request by quyenlq**

### CODE REVIEW

File: backend/benefit/applications/api/v1/serializers.py [hunk 1]
```diff
@@ -1124,10 +1125,19 @@ def _update_de_minimis_aid(self, application, de_minimis_data):
             ] = idx  # use the ordering defined in the JSON sent by the client
         serializer.save()
 
+    def _get_request_user_from_context(self):
+        request = self.context.get("request")
+        if request:
+            return request.user
+        return None
+
     def logged_in_user_is_admin(self):
```
Comment (klemmari1) [inline backend/benefit/applications/api/v1/serializers.py:1134]: Is / will this be used somewhere?

File: backend/benefit/applications/api/v1/serializers.py [hunk 2]
```diff
@@ -1124,10 +1125,19 @@ def _update_de_minimis_aid(self, application, de_minimis_data):
...
-        # TODO: user management
+        user = self._get_request_user_from_context()
+        if user:
+            return user.is_handler()
         return False
 
     def get_logged_in_user_company(self):
```
Comment (klemmari1) [inline backend/benefit/applications/api/v1/serializers.py:1140]: Is / will this be used somewhere?

**Review by klemmari1 — APPROVED** [review]
Left a couple of questions but looks good to me!

File: backend/benefit/applications/api/v1/serializers.py (hunk 2 above)
Comment (quyenlq) [reply backend/benefit/applications/api/v1/serializers.py:1140]: Yes, they are already used here: https://github.com/City-of-Helsinki/yjdh/pull/261/files#diff-91a79cb2d83133bd53a5d3272d60fe0e61c2bdc63af91389490d7cf661cbb849R1085. But they were only place-holder previously.

File: backend/benefit/users/utils.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from shared.oidc.utils import get_organization_roles
+
+
+def get_business_id_from_user(user):
```
Comment (haavikko) [inline backend/benefit/users/utils.py:4]: Good utility function

File: backend/benefit/applications/api/v1/serializers.py [hunk 4]
```diff
@@ -1311,10 +1312,19 @@ def _update_de_minimis_aid(self, application, de_minimis_data):
             ] = idx  # use the ordering defined in the JSON sent by the client
         serializer.save()
 
+    def _get_request_user_from_context(self):
```
Comment (haavikko) [inline backend/benefit/applications/api/v1/serializers.py:1315]: This same function will be useful in other serializers too

**Review by haavikko — APPROVED** [review]
Looks good to me :-)

**Pull request merged**